src/sympc/protocol/spdz/spdz.py

```diff
@@ -81,18 +81,22 @@ def mul_parties(
 
     op = getattr(operator, op_str)
 
-    eps_b = op(eps, b_share)
-    delta_a = op(delta, a_share)
+    eps_b = op(eps, b_share.tensor)
```

Comment:
this was a bit tricky to spot -> `op(eps, b_share)` would trigger `rmatmul` so   `op(b_share,eps)` which doesn't work for matmul because of shape mismatch